resource_manager/google/cloud/resource_manager/project.py

```diff
@@ -85,6 +86,8 @@ def set_properties_from_api_repr(self, resource):
         self.number = resource['projectNumber']
         self.labels = resource.get('labels', {})
         self.status = resource['lifecycleState']
+        if 'parent' in resource:
+            self.parent = resource['parent']
```

Comment:
You aren't overwriting, the resource sometimes gets assigned a parent and you need to include that parent when you update it back.